requirements.txt

```diff
@@ -10,3 +10,7 @@ fasttext-wheel
 pandas
 urllib3
 cloudpathlib[s3, gs, azure]
+fastapi[all]
+uvicorn
```

Comment:
I think the four new requirements should be more in the extra requirements and be in an api_requirements.txt for developers, and rather be downloaded on the side.

Reply:
I see the point, should the app be in another package from deepparse then ? And it has deepparse package as a dependency ?

Reply:
Otherwise people can and will try to import the app without the proper packages installed

Reply:
No, we just need to add a try/catch in the app's import and raise a message if the imports are not there.

Reply:
Look at colorama import.

Reply:
After a conversation with @davebulaval, we concluded that the best way to import the app dependencies were to pass them as extra requirements, this way the app dependencies can be installed using `pip install deepparse[app]`